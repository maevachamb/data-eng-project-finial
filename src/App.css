.App {
  text-align: center;
  background: #eee;
}

#regForm {
  background-color: #ffffff;
  margin: 0px auto;
  margin-top : 25px;
  font-family: Raleway;
  padding: 40px;
  border-radius: 10px;
  width: 40%;
}

#register {
  color: #2a4f798c;
  font-weight: bolder;
  margin-bottom:10%;
}

#submit_{
  width: 50%;
  text-align: center;
  margin-top: 5%;
}
#submit_:active{background-color:#2a4f798c;}

#result {
  border-radius: 25px;
  background: #7373732f;
  padding: 20px;
  margin: auto;
  margin-top : 25px;
  width: 500px;
  height: auto;
}

body {
  background: #eee
}

h1 {
  text-align: center;
  font-size: 50px;
}
h4 {
  text-align: left;
  font-weight: lighter;
}

input {
  padding: 10px;
  width: 100%;
  font-size: 17px;
  font-family: Raleway;
  border: 1px solid #aaaaaa;
  border-radius: 10px;
  -webkit-appearance: none
}

.tab input:focus {
  border: 1px solid #2a4f798c !important;
  outline: none
}

.tab {
  display: inline;
}