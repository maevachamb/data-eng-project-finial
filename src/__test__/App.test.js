import { render, screen } from '@testing-library/react';
import App from '../App';

test('renders learn react link', () => {
  const web = render(<App />);
  expect(web).toMatchSnapshot();
});
