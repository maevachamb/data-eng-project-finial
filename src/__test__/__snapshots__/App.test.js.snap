// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div>
          <div
            class="App"
          >
            <form
              id="regForm"
            >
              <h1
                id="register"
              >
                The Toxicity Monitor
              </h1>
              <div
                class="tab"
              >
                <h4>
                  The Toxicity Monitor is a toxicity analysis application,which, given a piece of text, is able to determine if the text is toxic or not.
                  <br />
                  <br />
                  <b>
                    Try it by writing something in english below.
                  </b>
                </h4>
                <input
                  id="user_text"
                  name="user_text"
                  placeholder="Please write something here..."
                  type="text"
                  value=""
                />
                <br />
              </div>
              <input
                id="submit_"
                type="submit"
                value="Send"
              />
            </form>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div>
        <div
          class="App"
        >
          <form
            id="regForm"
          >
            <h1
              id="register"
            >
              The Toxicity Monitor
            </h1>
            <div
              class="tab"
            >
              <h4>
                The Toxicity Monitor is a toxicity analysis application,which, given a piece of text, is able to determine if the text is toxic or not.
                <br />
                <br />
                <b>
                  Try it by writing something in english below.
                </b>
              </h4>
              <input
                id="user_text"
                name="user_text"
                placeholder="Please write something here..."
                type="text"
                value=""
              />
              <br />
            </div>
            <input
              id="submit_"
              type="submit"
              value="Send"
            />
          </form>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
